src/labels/CustomLabels.labels

```diff
@@ -4980,6 +4980,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>stgAddrMustChooseClass</shortDescription>
         <value>You must choose a verification service in order to enable automatic verification.</value>
     </labels>
+    <labels>
+        <fullName>stgAddrPermissionDenied</fullName>
+        <categories>Settings, address</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Permission Denied on Address Verification Panel</shortDescription>
+        <value>You do not have permission to verify Address. Please contact your system administrator for more information.</value>
```

Comment:
@sam-knox , can you please advise on the message displayed when user has no permission to view Address Settings on NPSP Settings? The same message is displayed when user has no permission to verify addresses when Health Check is executed.

<img width="930" alt="screen shot 2018-12-06 at 10 30 06 am" src="https://user-images.githubusercontent.com/28743286/49604721-84be7700-f943-11e8-853d-8c4a5193c976.png">

<img width="938" alt="screen shot 2018-12-06 at 10 31 00 am" src="https://user-images.githubusercontent.com/28743286/49604736-89832b00-f943-11e8-8dab-ea50364c5301.png">